src/Input/test/InputStylesSpec.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import Input from '../index';
-import { getDOMNode, getStyle, toRGB, inChrome } from '@test/utils';
+import { getStyle, toRGB, inChrome } from '@test/utils';
 
 import '../styles/index.less';
 
 describe('Input styles', () => {
   it('Input should render the correct styles', () => {
     const instanceRef = React.createRef<HTMLInputElement>();
     render(<Input ref={instanceRef} />);
-    const dom = getDOMNode(instanceRef.current);
+    const dom = instanceRef.current as Element;
     inChrome &&
       assert.equal(getStyle(dom, 'border'), `1px solid ${toRGB('#e5e5ea')}`, 'Input border');
```

Comment:
Use `expect` instead of `assert` as the test assertion to maintain code style consistency.